### PR TITLE
Dance Form Classification

### PR DESCRIPTION
**Pull request by jahenvins**
# Pull Request for DL-Simplified 💡
Performed EDA on the given Dataset and created a CNN model for Dance form classification. Also Plotted training loss and validation loss to check the performance of the model

## Issue Title : Dance Form Classification 
- **Info about the related issue (Aim of the project)** : Create a DL model which will predict the dance forms.
- **Name:** Jahnevi N S
- **GitHub ID:**  jahenvins 
- **Email ID:** jahnevi@gmail.com 

Closes: #issue number that will be closed through this PR

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
Describe how have you verified the changes made

## Checklist: ☑️

- [ ] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Please follow the project structure and add your files accordingly.
@jahenvins

**Comment by abhisheks008** [comment]
@jahenvins please do the required changes otherwise this will not be counted towards the contribution to this project repository.
